src/data/customer/__test__/job.test.js

```diff
@@ -45,3 +46,119 @@ describe('Show job', async () => {
     }
   });
 });
+
+describe('job create', async () => {
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 90000;
+  let token;
+  let routePayload;
+  let route1;
+  let route2;
+  let routeIds;
+
+  beforeAll(async () => {
+    token = await getTokenAsync(
+      CONFIG.email,
+      CONFIG.password,
+      CONFIG.clientId,
+      CONFIG.clientSecret
+    );
+
+    routePayload = {
+      driverId: 10714,
+      pickupDate: CONFIG.jobTest.date,
+      route_settings: '{}',
+    };
+
+    // create route 1 now
+    const route1Res = await storeRouteAsync(
+      {
+        routes: [
+          {
+            ...routePayload,
+            routeLocations: [
+              {
+                sequence: 1,
+                orderId: CONFIG.jobTest.orderIds[0],
+                locationTypeId: 3,
```

Comment:
why do you have to pass locationTypeId ?

Reply:
@maartenhemmes it indicates what type of route location it is pickup or drop off.